examples/lighting-app/nrfconnect/README.md

```diff
@@ -166,8 +166,8 @@ using the following command:
 
         $ git submodule update --init
 
-The example requires the nRF Connect SDK v1.5.0. You can either install it along
-with the related tools directly on your system or use a Docker image that has
+The example requires having installed recommended version of the nRF Connect SDK.
```

Comment:
Maybe we should skip information about the version where if we can't provide a specific version?